yt_dlp/extractor/stripchat.py

```diff
@@ -49,7 +49,7 @@ def _real_extract(self, url):
         for host in traverse_obj(data, ('config', 'data', (
                 (('features', 'featuresV2'), 'hlsFallback', 'fallbackDomains', ...), 'hlsStreamHost'))):
             formats = self._extract_m3u8_formats(
-                f'https://b-{server}.{host}/hls/{model_id}/master/{model_id}_auto.m3u8',
+                f'https://b-{server}.{host}/hls/{model_id}/{model_id}.m3u8',
```

Comment:
remove the `server` var assignment above and:
```suggestion
                f'https://edge-hls.{host}/hls/{model_id}/master/{model_id}_auto.m3u8',
```

Reply:
That looks like that did the trick!